pkg/database/migrate.go

```diff
@@ -325,33 +316,7 @@ func CheckAndMigrateDB(migrate bool, db *gorm.DB) error {
 				// We need to run this migration without foreign key constraints to avoid
 				// the cascading delete to kick in and delete all the runs.
 				if err := runWithoutForeignKeyIfNeeded(func() error {
```

Comment:
this is part of the migration

Reply:
@jgiannuzzi will you look at the current implementation and see if that looks good?